mart/attack/callbacks/progress_bar.py

```diff
@@ -15,10 +15,11 @@
 class ProgressBar(Callback):
     """Display progress bar of attack iterations with the gain value."""
 
-    def on_run_start(self, adversary, input, target, model, **kwargs):
+    def on_run_start(self, adversary=None, **kwargs):
         self.pbar = tqdm.tqdm(total=adversary.max_iters, leave=False, desc="Attack", unit="iter")
```

Comment:
Instead of doing `adversary=None`, I would do:
```
def on_run_start(self, *, adversary, **kwargs):
```
That lets the reader know adversary is a required keyword argument.

Reply:
Thanks. I wasn't aware of such a feature in Python.

Reply:
Fixed.